master/usage/configuration/ip-in-ip.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configuring IP-in-IP
+---
+
+If your network fabric performs source/destination address checks 
+and drops traffic when those addresses are not recognized, it may be necessary to
+use IP-in-IP encapsulation of the inter-workload traffic.
+  
+This is often the case for public-cloud environments where you have limited control
+over the fabric and you are not peering your Calico BGP network with the rest of the
+network.
+
+Calico can be configured to use IP-in-IP encapsulation by enabling the IPIP option
+on the [IP pool resource]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool). 
+When enabled, Calico will use IP-in-IP encapsulation when routing packets *to*
+workload IPs falling in the IP pool range.
+
+An optional `mode` field toggles when IP-in-IP is used:
+-  A value of `always` is used when IP-in-IP is always required.  This is the default
+   value used if the `mode` field is omitted in the configuration.
+-  A value of `cross-subnet` is used when IP-in-IP is only required when routing 
+   between hosts on different subnets.
+   
+> **NOTE**
+>
+> Routing between Workloads and Host Endpoints running Calico is asymmetric when
```

Comment:
> it is because the traffic one direction is tunneled but in the other direction it is no

This should be _fine_ so long as nothing is dropping the traffic- whether that is in the underlying network, or an RPF on the hosts, or something else.